Documentation/subcommands/run.md

```diff
@@ -342,11 +342,14 @@ For more details see the [hacking documentation](../hacking.md).
 
 | Flag | Default | Options | Description |
 | --- | --- | --- | --- |
+| `--cap-remove` | none | capability to remove (example: '--cap-remove=CAP\_SYS\_CHROOT,CAP\_MKNOD') | Capabilities to remove from the process's capabilities bounding set, all others from the default set will be included |
```

Comment:
https://daringfireball.net/projects/markdown/syntax

> Markdown allows you to use backslash escapes to generate literal characters which would otherwise have special meaning in Markdown’s formatting syntax.

Underscores can be used for emphasis:

- _single underscores_
- __double underscores__

So here, it is making sure that underscores are not interpreted as emphasis.